### ISSUE TITLE
Help with Vendor-Specific Video code

### ISSUE THREAD

**Issue by SICKSAUCE** [balu100/dolby-vision-for-windows#2]
Hi ! 

I'm really sorry to bother you but I don't understand how to use the excel sheet at all... 
Could you explain me or help me get the correct hex string for my case ?

My Vendor-Specific Video code is 4d4e4a725a7776. It is a TCL C825.

Thanks !

**Comment by robb213** [comment]
I've been trying to figure it out too. In my case I'm working with an LG C2 (480a7e86607694).

I read around on https://discourse.coreelec.org/t/edid-override-injecting-a-dolby-vsvdb-block/51510?page=1, which is the domain @djnice linked the file from in the other thread. Digging around I can upload the exported CRU .bin file here: https://people.freedesktop.org/~imirkin/edid-decode/

This will feed us the EDID in a readable format.

Under Vendor-Specific Video Data Block I found some information. I can see: Interface: Standard + Low-Latency. That tells me my TV supports Std + LLDV. But some of it is confusing for me. I.e. "Supports 10b 12b 444: Not supported" Does this mean 10b is supported but 12b is not, or that neither is supported? Then there's other values for which I do not know what they are named in the EDID, like Backlight Ctrl, etc.

**Comment by robb213** [comment]
> Separately, I have another question: is it possible for this EDID hack to support other media players? I naively thought that it would work across the OS, but it seems to only work for Windows Media Player. I've tried Jellyfin Media Player and VLC, but neither are able to play DV.

I have the same results as you. Additionally, I installed the [Dolby Vision app from the MS Store](https://apps.microsoft.com/detail/9mvmz93n61t9?hl=en-us&gl=US) (via [AdGuard](https://store.rg-adguard.net/)). @sam-6174 Have you installed the app (latest `.appxbundle`) and noticed any changes?

**Comment by pitrako** [comment]
I think guardians of the galaxy doesn't have dolby vision on pc.

**Comment by robb213** [comment]
> I think guardians of the galaxy doesn't have dolby vision on pc. Also I used the python script on my PUS7304 (Philips TV) and I get this "Warning: `video_hex` of '459f4ba43b26bb' is already enabled with LLDV-HDMI", what this means that only supports low latency dolby vision? I want to enable TV-LED ("real dolby vision mode"), Is this possible with my TV?
> 
> EDIT: In the link robb213 posted (https://people.freedesktop.org/~imirkin/edid-decode/) says my tv supports Interface: Standard + Low-Latency + Low-Latency-HDMI.

Yes, my oversight with the game. The Dolby gaming page I was on was a landing page for their Dolby in games in general. While it seems easier to find games that support Atmos, I can't really find any lists that show DV for PC games. Battlefield 1 may, and Metro Exodus may as well, but only for XSX. Will try both and edit the post with an update.

**Comment by pitrako** [comment]
The only games that have dolby vision on pc are Mass Effect Andromeda, Need for speed Heat, Battlefield 1 and Anthem.

**Comment by robb213** [comment]
I tried out BF1. Switching to DV in-game appears to do nothing for me. The TV does not seem to reflect the mode either.

**Comment by ValeZAA** [comment]
> The only games that have dolby vision on pc are Mass Effect Andromeda, Need for speed Heat, Battlefield 1 and Anthem.

Anthem does not, and for others Nvidia removed support

**Comment by paolorafael** [comment]
Hi, my TV is hisense u7g, and running the python  script, i get this error:

SyntaxError: invalid decimal literal
>>> enable_dolby_vision_hdmi.py 4803628a546f7f
  File "<python-input-5>", line 1
    enable_dolby_vision_hdmi.py 4803628a546f7f

What do I do?

**Comment by balu100** [comment]
@paolorafael Could you try running the updated script? 

Current output:
```bash
python .\dvhdmi.py 4803628a546f7f
All tests passed.
Update video_hex from '4803628a546f7f' to '4803638a546f7f' to enable LLDV-HDMI

**Comment by paolorafael** [comment]
That's weird. I only downloaded the script from your repo.  https://github.com/balu100/dolby-vision-for-windows/blob/main/enable_dolby_vision_hdmi.py

**Comment by holingher** [comment]
@robb213 Hi, I have some issues with a LG C2, can you help me? I created another issue ticket into this repo, but I've seen you also have a C2.

**Comment by apoklyps3** [comment]
unfortunately this doesn't seem to do anything DV related while in Windows.
When playing DV on the TV side (e.g.  TCL google tv) , new picture modes are unlocked for DV and they have extra parameters that can be altered, including full local dimming on some TV's which is a must to enjoy this kind of content.
None of that is availabe in PC mode/Game mode

**Comment by b00zzy** [comment]
Hi, my TV is old (lg oled b6) and hex i get from EDID EDITOR is 24180900ad5244a9230c

Can it be converted?

**Comment by balu100** [comment]
it's been open for too long, closing,
